### PR TITLE
Update Welcome Wagon for Venueless

### PR DESCRIPTION
**Pull request by CollierCZ**
----
📚 Documentation preview 📚: https://writethedocs-www--2218.org.readthedocs.build/

### CODE REVIEW

File: docs/include/conf/virtual/welcome-wagon.rst [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-virtually meet you: feel free to reach out in Chat at the virtual
-conference if we can help make your time at the conference easier. You can also email us at {{ shortcode }}@writethedocs.org.
+We’re |WelcomeWagonNames| your Welcome Wagon, and we’re glad you’re attending Write the Docs!
+We realize a virtual conference might be totally new to you.
+This guide will help you navigate it,
+even if you’ve been to a Write the Docs conference before.
+
+Whether this is your first Write the Docs conference or your first virtual Write the Docs conference,
+we'll be here to help you make the most of it.
+We’d love to virtually meet you: feel free to send us a direct message at the virtual conference
```
Comment (mxsasha) [inline docs/include/conf/virtual/welcome-wagon.rst]:
```suggestion
We’d love to virtually meet you: feel free to ask anything in the #helpdesk channel at the virtual conference (not private) or send us a direct messsage
```

File: docs/include/conf/virtual/welcome-wagon.rst [hunk 2]
```diff
@@ -19,25 +22,25 @@ Where to get help
 Guides
 ~~~~~~
 
-You can prepare for the conference by reading through this guide. Don’t worry if you forget any of it – there will be a recording you can watch in the Helpdesk booth.
-Select **Expo** > **Write the Docs Helpdesk** to watch it anytime.
-
-You can also watch the `video directly on YouTube <https://www.youtube.com/watch?v=n8YXhL26YkE>`__.
+You can prepare for the conference by reading through this guide.
+Don’t worry if you forget any of it –
+we'll be waiting at the Helpdesk booth for any questions you might have.
```
Comment (mxsasha) [inline docs/include/conf/virtual/welcome-wagon.rst]:
```suggestion
we'll be waiting at the #helpdesk channel on Venueless for any questions you might have.
```

File: docs/include/conf/virtual/welcome-wagon.rst [hunk 3]
```diff
@@ -19,25 +22,25 @@ Where to get help
...
 
 Helpdesk
 ~~~~~~~~
 
 Have a look at the `FAQ section <#faqs>`__ we put together based on questions we got at previous conferences.
 
 If you still have questions, reach out the Welcome Wagon staff at the Helpdesk anytime.
-Select **Expo** > **Write the Docs Helpdesk** > **Chat** and enter your question.
+Find **Helpdesk** under *Channels*.
```
Comment (mxsasha) [inline docs/include/conf/virtual/welcome-wagon.rst]:
```suggestion
Find **#helpdesk** under *Channels*.
```

File: docs/include/conf/virtual/welcome-wagon.rst [hunk 4]
```diff
@@ -19,25 +22,25 @@ Where to get help
...
+If you don't see it there, select **Browse all channels** and join the **Helpdesk**.
 A member of the Welcome Wagon will be online during conference hours to help you.
 
 Message organizers
 ~~~~~~~~~~~~~~~~~~
 
 If you want to contact one of the conference organizers,
-go to the **People** page and search for an organizer's name.
+select **+Open a direct message** and search for an organizer's name.
```
Comment (mxsasha) [inline docs/include/conf/virtual/welcome-wagon.rst:45]:
```suggestion
select **+Open a direct message** and search for an organizer's name.
However, using the #helpdesk channel is preferred, as not all organizers are available all the time.
```

File: docs/include/conf/virtual/welcome-wagon.rst [hunk 5]
```diff
@@ -46,51 +49,55 @@ Slack
...
+To ask a question, during either the talk or the Q&A session:
 
-1. Open the **Chat** channel (on the right-hand side of the page).
-   Make sure it is the Chat for the Stage/session (not the Event chat).
-2. Enter your question prefixed with ``Q:``.
+1. In the Stage, open the **Questions** tab.
+2. Select **Ask a question**.
+3. Enter your question and select **Submit your question**.
 
 If there is time, a moderator will ask the question to the speaker.
```
Comment (mxsasha) [inline docs/include/conf/virtual/welcome-wagon.rst:90]:
```suggestion
If there is time, a moderator will ask the question to the speaker.
Note that the moderator might put their own twist on the questions or their order.
```

File: docs/include/conf/virtual/welcome-wagon.rst [hunk 6]
```diff
@@ -46,51 +49,55 @@ Slack
...
+
+View all questions asked and vote for those you find most interesting in the **Questions** tab.
 If your question isn’t asked live, chat with the speaker later and make a connection!
 
 Watching talks after the conference
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Don't worry if you miss something in a talk! Stage talks and Q&A sessions are recorded. Videos will be published in the `Write the Docs Video Archive <https://www.writethedocs.org/videos/>`__ shortly after the conference (typically about a week after the conference ends). To be notified when videos are published, `sign up for our newsletter <https://www.writethedocs.org/newsletter/>`__ or `subscribe to our YouTube channel <https://www.youtube.com/c/WritetheDocs/?sub_confirmation=1>`__.
+Don't worry if you miss something in a talk!
+Stage talks and Q&A sessions are recorded.
```
Comment (mxsasha) [inline docs/include/conf/virtual/welcome-wagon.rst]:
```suggestion
Stage talks, Q&A sessions, and lightning talks are recorded.
```

File: docs/include/conf/virtual/welcome-wagon.rst [hunk 7]
```diff
@@ -105,155 +112,192 @@ Use these lessons for attending Writing Day, Unconference, and sponsor sessions.
...
+- At the time the session starts,
+  find the table under *Channels* that corresponds to the one in the `Unconference schedule <{{unconf.url}}>`__.
 {% else %}
-- Starting Monday morning, check the Unconference schedule to see if there are any sessions you are interested in joining. New ones are added all the time, so check back periodically.
-- At the time the session starts, go to **Sessions** and select the session with the table number that corresponds to the one in the Unconference schedule.
+- At the time the session starts,
+  ind the table under *Channels* that corresponds to the one in the Unconference schedule.
 {% endif %}
-- The session leader will begin when the group has gathered.
+- The session leader will begin the session when the group has gathered.
```
Comment (mxsasha) [inline docs/include/conf/virtual/welcome-wagon.rst:238]:
```suggestion
- The session leader will begin the session when the group has gathered.
- You do not need to sign up anywhere to join a session.
```

File: docs/include/conf/virtual/welcome-wagon.rst [hunk 8]
```diff
@@ -105,155 +112,192 @@ Use these lessons for attending Writing Day, Unconference, and sponsor sessions.
...
+- A lightning talk is a five-minute talk
+  where you quickly share a concept or bit of info you find interesting.
+  Lightning talks are a great way to practice public speaking,
+  get people excited about your Unconference session,
+  and test interest in a conference proposal idea.
 - If you are interested in giving a lightning talk, be prepared!
   Read our `guide to lightning talks <https://www.writethedocs.org/conf/{{ shortcode }}/{{ year }}/lightning-talks/>`__.
-- Watch the conference chat and **Reception** for an announcement that Lightning Talk sign-ups have opened and for directions on how to sign up.
+- Watch the conference chat and your browser window
+  for an announcement that Lightning Talk sign-ups have opened and for directions on how to sign up.
```
Comment (mxsasha) [inline docs/include/conf/virtual/welcome-wagon.rst:255]:
```suggestion
- Lightning talks will open on Sunday at the start of Writing Day,
  close a while before the first session to allow us to select talks, and then reopen right after the Monday Lightning 
  Talk session.
  Watch the conference chat and your browser window
  for an announcement that Lightning Talk sign-ups have opened and for directions on how to sign up.
```

**Review by mxsasha — COMMENTED** [review]
This is great work! I left a bunch of inline suggestions, but they're pretty minor.

I do notice this page isn't currently in the top bar nav, I can fix that later.

File: docs/include/conf/virtual/welcome-wagon.rst [hunk 9]
```diff
@@ -1,12 +1,15 @@
...
-conference if we can help make your time at the conference easier. You can also email us at {{ shortcode }}@writethedocs.org.
+We’re |WelcomeWagonNames| your Welcome Wagon, and we’re glad you’re attending Write the Docs!
+We realize a virtual conference might be totally new to you.
+This guide will help you navigate it,
+even if you’ve been to a Write the Docs conference before.
+
+Whether this is your first Write the Docs conference or your first virtual Write the Docs conference,
+we'll be here to help you make the most of it.
+We’d love to virtually meet you: feel free to send us a direct message at the virtual conference
+if we can help make your time at the conference easier.
```
Comment (CollierCZ) [inline docs/include/conf/virtual/welcome-wagon.rst]:
```suggestion
We’d love to virtually meet you:
feel free to ask anything in the #helpdesk channel at the virtual conference (not private)
or send us a direct message
if we can help make your time at the conference easier.
```

File: docs/include/conf/virtual/welcome-wagon.rst (hunk 4 above)
Comment (CollierCZ) [reply docs/include/conf/virtual/welcome-wagon.rst:45]:
```suggestion
select **+Open a direct message** and search for an organizer's name.
It's best to use the #helpdesk channel as not all organizers are available all the time.
```

File: docs/include/conf/virtual/welcome-wagon.rst (hunk 8 above)
Comment (CollierCZ) [reply docs/include/conf/virtual/welcome-wagon.rst:255]:
```suggestion
- Sign-ups for Lightning Talks will open on Sunday at the start of Writing Day,
  close a while before the first session to allow us to select talks,
  and then reopen right after the Monday Lightning Talk session.
- Watch the conference chat and your browser window
  for an announcement that Lightning Talk sign-ups have opened and for directions on how to sign up.
```

**Review by mxsasha — APPROVED** [review]

**Pull request merged**